src/formatter.js

```diff
@@ -0,0 +1,72 @@
+const MESSAGE = Symbol.for('message');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyObject = require('yamlify-object');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyColors = require('yamlify-object-colors');
+
+const utils = require('./utils');
+const Message = require('./message');
+const Colorizer = require('./colorizer');
+const defaultErrorColors = require('./colors');
+
+const { format } = require('winston');
+
+module.exports = format((info, options = {}) => {
+  options = Object.assign(
+    {},
```

Comment:
Because the defaults below are in an object literal, we don't need to have an empty object literal here.

Reply:
True, but this is safer, there's no way for us to modify the object passed in.

First parameter is what everything is merged into, and since it's passed by reference I always err on the side of caution

Reply:
Oh shit you mean because the second one w literal. Yea yea. Could do. I didn't look at the diff close enough. I'll change it I change anything else.